src/intervals/conversion_promotion.jl

```diff
@@ -19,3 +11,103 @@ promote_rule{T<:Real, S<:Real}(::Type{Interval{T}}, ::Type{S}) =
 
 promote_rule{T<:Real}(::Type{BigFloat}, ::Type{Interval{T}}) =
     Interval{promote_type(T, BigFloat)}
+
+
+## Conversion rules
+
+# Just specifying Interval defaults to Float64 interval
+# Remove this default so that it is necessary to explicitly specify?
+convert(::Type{Interval}, x::Real) = convert(Interval{Float64}, x)
```

Comment:
What you are suggesting for rationals and `BigFloat` is already done by the constructor:

```
julia> x = 1//3
1//3

julia> Interval(x)
[1//3, 1//3]
```

You only need to explicitly call `convert` to convert it to some other type of interval:
```
julia> convert(Interval{Float64}, x)
[0.3333333333333333, 0.33333333333333337]
```
In that case, you would want to be explicit about the type you were converting to, I feel.